CONTRIBUTING.md

```diff
@@ -79,6 +79,7 @@ optimize:
 When Kibana runs in development mode it will automatically use bundled SSL certificates. These certificates won't be trusted by your OS by default which will likely cause your browser to complain about the cert. You can deal with this in a few ways:
 
   1. Supply your own cert using the `config/kibana.dev.yml` file.
+    - You can point at the development certificate found in `test/dev_certs` 
```

Comment:
I've just set up the environment and an alternative way of finding the cert is just looking it up in the directory and double clicking, without having to follow the other steps